cytopipe/cli/cli_docs.py

```diff
@@ -39,7 +46,15 @@
 Required Arguments:
     -d, --data          List of plate data files
     -m, --metadata      Path to metadata directory
-    -b, --barcode       Path to file containing barcode labeling
+
+Optional Arguments:
+    -b, --barcode       Path to file containing barcode labeling. This is used
+                        for cell morphology reads obtained by Cell Profiler.
```

Comment:
```suggestion
                        for cell morphology reads obtained by CellProfiler.
```

CellProfiler is one word too